cli/src/commands/extensions/api.rs

```diff
@@ -187,13 +187,14 @@ async fn get_package_details(
 /// Equivalent to `phylum parse`.
 #[op]
 async fn parse_lockfile(
-    op_state: Rc<RefCell<OpState>>,
+    _op_state: Rc<RefCell<OpState>>,
```

Comment:
I _think_ we could get away with just omitting `op_state` altogether here. IIRC it is passed in only if the `#[op]` proc macro finds it among the arguments to the op.

Reply:
The permissions still need to be validated here, so that's an open TODO. That's why I haven't removed it yet, it'll likely come back.